app/controllers/responses_controller.rb

```diff
@@ -4,9 +4,9 @@ def show
     render json: @response
   end
 
-  def letter
-    render json: Letter.find(params[:id]).responses
-  end
+  # def letter
+  #   render json: Letter.find(params[:id]).responses
```

Comment:
Are you planning on adding this back later? Or did you intend to remove it entirely? 

In general I would try and remove commented out code, unless you leave a comment explaining why it should be left in.

Reply:
Going to assume that this should be removed and update it myself